consensus/ConsensusFileGenerator.py

```diff
@@ -246,3 +265,9 @@ def generate_consensus_files(self):
         comments_file.close()
         progress_bar.finish()
         return consensus_file_name, comments_file_name
+
+    @staticmethod
+    def _check_alternative_history(transcript, c_dna, gene, export):
```

Comment:
what is the purpose of `gene`? `transcript:c_dna` should be unique here?

Reply:
On very rare occasions one variant can be assigned to two different genes.